CHANGELOG.md

```diff
@@ -1,4 +1,59 @@
-## v1.0.0
+## v1.0.0b3
+
+### Improvements
+- Add a configuration default user configurable through `verdi config` [[#2734]](https://github.com/aiidateam/aiida_core/pull/2734)
+- Try multiple ways to get default path for `verdi calcjob [in/out]putcat` [[#2615]](https://github.com/aiidateam/aiida_core/pull/2615)
+- Raise when returning an unstored `Data` from a `WorkflowNode` [[#2747]](https://github.com/aiidateam/aiida_core/pull/2747)
```

Comment:
```suggestion
- Raise when returning an unstored `Data` from a `WorkflowNode`, since this breaks data provenance. [[#2747]](https://github.com/aiidateam/aiida_core/pull/2747)
```